cmp/tab/TabModel.ts

```diff
@@ -170,4 +173,39 @@ export class TabModel extends HoistModel {
 
         this.disabled = disabled;
     }
+
+    //------------------
+    // Implementation
+    //------------------
+    private parseContent(content: Content | TabContainerConfig | TabConfig[]): Content {
+        // Recognize content is a child container spec and auto create with appropriate defaults
+        let raw = content as any,
+            childConfig: TabContainerConfig = null;
+        if (isArray(raw) && raw[0]?.content) {
+            childConfig = {tabs: raw};
+        } else if (raw.tabs) {
+            childConfig = raw;
+        }
+        if (childConfig) {
+            throwIf(XH.isMobileApp, 'Child Tabs not supported for Mobile TabContainer');
+
+            let {containerModel, id} = this,
+                modelRef = (this._childContainerModelRef = createObservableRef());
+            return tabContainer({
+                modelConfig: {
+                    renderMode: containerModel.renderMode,
+                    refreshMode: containerModel.refreshMode,
+                    emptyText: containerModel.emptyText,
+                    switcher: containerModel.switcher,
+                    track: containerModel.track,
+                    route: `${containerModel?.route}.${id}`,
```

Comment:
If `route` is not specified on the `containerModel`, I believe this should be omitted here entirely (currently would result in `"undefined.${id}"`